src/utils/api/socket.test.js

```diff
@@ -0,0 +1,47 @@
+import socket from './socket';
+// import { createServer } from 'http';
+// import { Server } from 'socket.io';
+
+jest.useRealTimers();
+
+describe('demo', () => {
+  // let server;
+  // let io;
+
+  // beforeAll((done) => {
+  //   server = createServer((req, res) => {
+  //     res.write('ok');
+  //     res.end();
+  //   });
+  //   io = new Server(server);
+  //   server.listen(() => {
+  //     const port = server.address().port;
+  //     socket.create({ baseUrl: `http://localhost:${port}` });
+  //     // socket.client.on('connect', done);
+  //     socket.client.on('connect', done);
+  //   });
+  // });
+  //
+  // afterAll(async () => {
+  //   await server.close();
+  //   await io.close();
+  // });
```

Comment:
can you remove this commented code if its not being used?

Reply:
no i want to keep it we had issues to running this it need to be fix later